spec/06-expressions.md

```diff
@@ -329,8 +329,8 @@ arguments, the following conditions must hold.
 - The names $x_i$ of all named arguments are pairwise distinct and no named
   argument defines a parameter which is already specified by a
   positional argument.
-- Every formal parameter $p_j:T_j$ which is not specified by either a positional
```

Comment:
I was hoping the language here expresses: each parameter has to have a corresponding argument. That argument could be supplied explicitly (for which there are two flavors of syntax, positional and named args) or implicitly (because the parameter list is marked implicit and an implicit was found). If an argument was not supplied in this way, it must have a default arg.

That's how I read, "every param which is not specified [in these ways] [must have] a default argument."

This is just to clarify that defaults are used only if an (explicit or implicit) arg is not specified. Also, trying to use the minimum of words.